### PR TITLE
This works WIP

### PR DESCRIPTION
**Pull request by gilacost**
TODO
- [x] change users

### CODE REVIEW

**Comment by gilacost** [comment]
had to mv /etc/shells
had to mv /etc/nix/nix.conf several times

File: darwin-configuration.nix [hunk 1]
```diff
@@ -1,54 +1,28 @@
...
+  programs.bash.enable = false;
 
-  # Nix
-  services.nix-daemon.enable = true;
+  programs.zsh.enable = true;
 
   nix = {
-    trustedUsers = [ "root" "lehoff" ];
-    package = pkgs.nix_2_4;
+    trustedUsers = [ "root" "pepo" ];
```
Comment (gilacost) [inline darwin-configuration.nix]:
```suggestion
    trustedUsers = [ "root" "lehoff" ];
```

File: flake.nix [hunk 2]
```diff
@@ -17,15 +17,15 @@
         {
           home-manager.useGlobalPkgs = true;
           home-manager.useUserPackages = true;
-          home-manager.users.lehoff = import ./home.nix;
+          home-manager.users.pepo = import ./home.nix;
```
Comment (gilacost) [inline flake.nix]:
```suggestion
          home-manager.users.lehoff = import ./home.nix;
```

File: flake.nix [hunk 3]
```diff
@@ -17,15 +17,15 @@
...
         }
       ];
     in {
 
       darwinConfigurations = {
 
         "mimer" = darwin.lib.darwinSystem {
-          system = "aarch64-darwin";
+          system = "x86_64-darwin";
```
Comment (gilacost) [inline flake.nix]:
```suggestion
          system = "aarch64-darwin";
```

File: home.nix [hunk 4]
```diff
@@ -1,57 +1,66 @@
...
+    enableCompletion = true;
+    enableAutosuggestions = true;
 
+    history.extended = true;
+
+    # Review prezto and pure options
     prezto = {
       enable = true;
-      prompt.theme = "bart";
+      prompt.theme = "pure";
```
Comment (gilacost) [inline home.nix]:
```suggestion
      prompt.theme = "bart";
```

**Review by lehoff — APPROVED** [review]

**Pull request merged**